packages/babel-parser/test/fixtures/experimental/explicit-resource-management/invalid-using-binding-await/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sourceType": "script"
```

Comment:
Can you also add a test in modules? It throws that `await` is reserved in modules, right?